project_milestone/views/project.xml

```diff
@@ -0,0 +1,83 @@
+<!-- License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+   <!-- List View-->
+   <record id="project_milestone_view_inherit_list" model="ir.ui.view">
+     <field name="name">project.milestone.tree</field>

```

Comment:
Please indent properly ***all the file*** with 4 space tab stops, it's OCA standard.

Reply:
I was having problems with my editor and how it was displaying spaces but I think got the correct settings. The code should be up to standard now. Thanks